### PR TITLE
Ref #4220: Upgrade to SRU2022-10.0.1

### LINKED ISSUE #4220: Camel 3.20 - camel-swift extension
There is a new component (dataformat)
https://github.com/apache/camel/tree/main/components/camel-swift

### PR DESCRIPTION
**Pull request by essobedo**
fixes #4220 (part 2)

## Motivation

After migrating to Upgrade to SRU2022-10.0.0, some tests failed due to [a reflection issue when serializing the MX messages in JSON ](https://github.com/prowide/prowide-iso20022/pull/83#issuecomment-1580150900).

## Modifications

* Removes the temporary workaround

### CODE REVIEW

**Comment by essobedo** [comment]
Depends on https://github.com/apache/camel/pull/10398

**Review by oscerd — APPROVED** [review]

**Comment by JiriOndrusek** [comment]
@essobedo Hi, I used `git push --force-with-lease`, which is recommended procedure for maintenance of `camel-main`. I'm very sorry that I missed your PR and sorry for any inconvenience it caused to you. Usually after `git push --force-with-lease` I should comment all existing PRs and point out with a comment, that this is happening and rebase is needed.
(I'm usually pushing changes each day in case the build of `camel-main` fails with the recent changes in Camel)

**Comment by essobedo** [comment]
Actually, you should only push your new commits no need to force them.

**Comment by JiriOndrusek** [comment]
Oh, I can do that, sorry for doing it wrong. I might miss-understand use of `git push --force-with-lease` for maintenance. Thanks for telling me! 

But 

I just checked again `git push --force-with-lease`. It's like a normal push, which pushes only in case, that the there is no new commit in the repo.
(I'm using it to ensure that I do not overwrite any commit which was merged/pushed in meantime)
Therefore if I push with force-with-lease something it behaves similarly to `push` alone. If I yesterday pushed the change, the problem would be the same, right?
In other words If I'd have used `git push` the yesterday, the result would be the same from you POV, right?

**Comment by JiriOndrusek** [comment]
Perhaps my problem yesterday was, that I forgot to push in the morning and did it later that day.

Sorry for discussing this topic in so many comments. I'm trying understand what to do correctly and why.

**Comment by essobedo** [comment]
If it is clear to you now, that's good news, otherwise don't hesitate to reach out to me on Zulip
